<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#1C1C1B</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="generators">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {color: #429F42 ; background-color: #2D2D2D; border: none;}
QListWidget:hover {color: #5BD75B ; background-color: #3A3A39; border: none;}</string>
    </property>
    <property name="movement">
     <enum>QListView::Static</enum>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="solve">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color: #429F42 ; background-color: #2D2D2D; border: none;}
 QPushButton:hover {color: #5BD75B ; background-color: #3A3A39; border: none;} 
QPushButton:pressed {color: #6CFF6C ; background-color: #474746; border: none;}</string>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="power">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {color: #429F42 ; background-color: #2D2D2D; border: none;} 
QLineEdit:hover {color: #5BD75B ; background-color: #3A3A39; border: none;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Energy per second</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="solution">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>521</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser {color: #429F42 ; background-color: #2D2D2D; border: none;}
QTextBrowser:hover {color: #5BD75B ; background-color: #3A3A39; border: none;}</string>
    </property>
   </widget>
   <widget class="QLabel" name="solution_text">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>2</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: #D1D152}</string>
    </property>
    <property name="text">
     <string>Solution:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ru_button">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color: #429F42 ; background-color: #2D2D2D; border: none;}
 QPushButton:hover {color: #5BD75B ; background-color: #3A3A39; border: none;} 
QPushButton:pressed {color: #6CFF6C ; background-color: #474746; border: none;}</string>
    </property>
    <property name="text">
     <string>RU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="en_button">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {color: #429F42 ; background-color: #2D2D2D; border: none;}
 QPushButton:hover {color: #5BD75B ; background-color: #3A3A39; border: none;} 
QPushButton:pressed {color: #6CFF6C ; background-color: #474746; border: none;}</string>
    </property>
    <property name="text">
     <string>EN</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="round">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton {color: #429F42}</string>
    </property>
    <property name="text">
     <string>Round</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="selected_type">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {color: #429F42 ; background-color: #2D2D2D; border: none;}\nQTextBrowser:hover {color: #5BD75B ; background-color: #3A3A39; border: none;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
